src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.exception.AppealNotFoundException;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.CreatedBy;
+import uk.gov.companieshouse.repository.AppealRepository;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class AppealService {
+
+    private final AppealRepository appealRepository;
+
+    public String saveAppeal(String companyId, Appeal appeal, String userId) throws Exception {
```

Comment:
Since company number is in the appeal there is no need to pass it as separate arg. Please remove.